build/pkgs/primecountpy/spkg-configure.m4

```diff
@@ -0,0 +1,5 @@
+SAGE_SPKG_CONFIGURE([primecountpy], [
+  SAGE_SPKG_DEPCHECK([cysignals primecount], [
```

Comment:
Not necessarily for this PR, but a possible next refinement:
`primecount` is only used via `primecountpy`; so instead of listing it in `SAGE_SPKG_DEPCHECK`, we could mark `primecount` as "not required" when `primecountpy` is available -- like we do for `virtualenv` as the dependency of only `tox`.

Reply:
how do we do this? where is this "not required" thing should be?

Reply:
You can see an example in e.g. `filelock/spkg-configure.m4`. In theory we could do this for any package but it requires hard-coding the entire dependency tree in m4 and even I am not in a rush to do that. This is a relatively easy case though.